scripts/create-docker-context-for-node-component.js

```diff
@@ -32,7 +32,8 @@ const argv = yargs
         repository: {
             description:
                 "The repository to use in auto tag generation. Will default to '', i.e. dockerhub unless --local is set. Requires --tag=auto",
-            type: "string"
+            type: "string",
+            default: process.env.MAGDA_DOCKER_REPOSITORY
         },
         name: {
```

Comment:
It'd be nice if we could add env var support to `name` parameter as well.
Currently, most modules have default name `data61/xxxxx`. 
It works for most docker repo.
But for repo like github registry, it cares about pushing repository url pattern and the  `data61/` part make it complaint.